news/php/news_articles_archive.php

```diff
@@ -109,6 +111,21 @@ function inspect_news_archive_page($xml, $categories){
             $options = array('school', 'topic', 'department', 'adult-undergrad-program', 'graduate-program', 'seminary-program', 'unique-news');
             $page_info['display-on-feed'] = match_metadata_articles($xml, $categories, $options, "news");
         }
+    } else {
+        # if its not the president announcements page, we want to make sure that the prez announcements are NOT shown.
+        foreach ($xml->{'dynamic-metadata'} as $md) {
+            $name = $md->name;
+
+            foreach ($md->value as $value) {
+                if (strtolower($value) == "none" || strtolower($value) == "select") {
```

Comment:
Why do you need to check this?

Reply:
ah, i suppose I don't. I copy-pasted this from some other code that had this check built in. I'll commit this out